src/accounts.js

```diff
@@ -307,33 +353,71 @@ async function createKeyPair(password, salt, authAccountName, permissionName, op
 
   if (options.confirm) {
     const awaitTransactionOptions = getAwaitTransactionOptions(options);
-    await this.awaitTransaction(() => addPermission.bind(this)(authAccountName, [keys.publicKeys.active], permissionName, parentPermission, options), awaitTransactionOptions);
+    await this.awaitTransaction(() => addPermission.bind(this)(authAccountName, [keys.publicKeys.active], permissionName, parentPermission, false, options), awaitTransactionOptions);
   } else {
-    await addPermission.bind(this)(authAccountName, [keys.publicKeys.active], permissionName, parentPermission, options);
+    await addPermission.bind(this)(authAccountName, [keys.publicKeys.active], permissionName, parentPermission, false, options);
   }
 
   return keys;
 }
 
 async function createBridgeAccount(password, salt, authorizingAccount, options) {
```

Comment:
We could use a comment here at least - not clear that options.oreAccountName is optional and if provided will be the new account (or generates one otherwise)

Reply:
for createKeyPair, it is not expecting options.oreAccountName as the permission name coming in is not an account and doesn't exist on chain ever. So oreid passes in a permissionName and that gets added to the service account for the app